app/models/efile_submission.rb

```diff
@@ -117,10 +117,8 @@ def previously_transmitted_submission
     previous_submission if previous_submission.last_transition_to("transmitted").present?
   end
 
-  def generate_irs_address
-    if address.present? && address.skip_usps_validation
-      return OpenStruct.new(valid?: true)
-    end
+  def generate_verified_address
+    return verified_address if verified_address.present?
```

Comment:
the active record object will respond to .valid?

Reply:
yeah, ill revert to using an openstruct because i dont want there to be unexpected return values if we end up adding validations that can fail on addresses at some point or something like that